packages/components/psammead-media-indicator/src/index.stories.jsx

```diff
@@ -100,3 +100,28 @@ storiesOf('Components|MediaIndicator/Photo', module)
     ),
     { notes },
   );
+
+storiesOf('Components|MediaIndicator/Index Alsos', module)
```

Comment:
Instead of creating another story, could we have a `boolean` knob on `indexAlsos`?

Reply:
Sound good to me, but I will also have to add new ones for top story without duration.